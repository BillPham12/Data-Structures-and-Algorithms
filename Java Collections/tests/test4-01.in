hello
asdasd
asd
asd
as
das
das
d
asd
as
12312
 312
 





12
31
23
123
123
123
12
312
3
123
123
123
12
312
312
312
312
