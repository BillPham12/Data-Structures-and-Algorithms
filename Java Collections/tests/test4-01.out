hello
asdasd
asd
asd
as
das
das
d
asd
as
12312
 312
 





12
31
23
